pkg/sdk/helper.go

```diff
@@ -112,14 +122,24 @@ func (h *SDKHelper) API(serviceModelName string) (*SDKAPI, error) {
 		// Calling API.ServicePackageDoc() ends up resetting the API.imports
 		// unexported map variable...
 		_ = api.ServicePackageDoc()
-		return &SDKAPI{api, nil, nil, h.APIGroupSuffix}, nil
+		sdkapi := &SDKAPI{
+			API:            api,
+			APIGroupSuffix: h.APIGroupSuffix,
+			CustomShapes:   []*CustomShape{},
+			opMap:          nil,
+			typeRenames:    nil,
+		}
+
+		h.InjectCustomShapes(sdkapi)
```

Comment:
From the current implementation of Helper, I don't think it makes sense for one of its methods to be able to change/edit `sdkapi`.

Is there any reason this couldn't be a setter? ex: `func (a *SDKAPI) SetCustomShapes(cfg ackgenconfig.Config)`